addons/ai/XEH_preInit.sqf

```diff
@@ -6,4 +6,16 @@ PREP_RECOMPILE_START;
 #include "XEH_PREP.hpp"
 PREP_RECOMPILE_END;
 
+// Fix rotating garrisoned units
+["ModuleCurator_F", "Init", {
+    params ["_logic"];
+
+    _logic addEventHandler ["CuratorObjectEdited", {
+        params ["", "_object"];
+        if (_object isKindOf "Man" && {_object getVariable [QGVAR(garrisoned), false]}) then {
```

Comment:
Is the `isKindOf` check even required? It's not like you can garrison vehicles or planes.